src/opt/test/test_renderer.cc

```diff
@@ -79,23 +89,33 @@ void TestRendererPixelAccuracy(const Camera& camera) {
   pcl::toPCLPointCloud2(mesh_vertex_cloud, polygon_mesh.cloud);
   
   // Write faces into mesh.
-  int num_faces = 2 * (camera.width() - 1) * (camera.height() - 1);
+  int num_faces = 2 * (grid_width - 1) * (grid_height - 1);
   polygon_mesh.polygons.reserve(num_faces);
-  for (int y = 0; y < camera.height() - 1; ++ y) {
-    for (int x = 0; x < camera.width() - 1; ++ x) {
+  for (int y = 0; y < grid_height - 1; ++y) {
+    for (int x = 0; x < grid_width - 1; ++x) {
+      const int topLeftId = x + y * grid_width;
```

Comment:
Minor: This naming might be a little confusing since these are not arbitrary IDs, but rather successive indices. Would thus call them indices (which is a standard term in computer graphics) instead of IDs.